tensorflow/python/keras/layers/normalization.py

```diff
@@ -946,6 +947,24 @@ def __init__(self,
 
     self.supports_masking = True
 
+    self.fused = False
```

Comment:
Make this private: self._fused. And you should initialize this to None instead of False, since it will be updated in build(). Also add a brief comment, such as "Indicates whether a faster fused implementation can be used. This will be set to True or False in build()".